testsuite/integration/microprofile-tck/rest-client/pom.xml

```diff
@@ -441,7 +441,7 @@
                         <jboss.inst>${basedir}/target/wildfly</jboss.inst>
                         <jboss.install.dir>${basedir}/target/wildfly</jboss.install.dir>
                         <jboss.server.config.file.name>standalone-microprofile.xml</jboss.server.config.file.name>
-                        <microprofile.jvm.args>${microprofile.jvm.args} -Dorg.eclipse.microprofile.rest.client.tck.timeoutCushion=30 -Dorg.wildfly.unsupported.skip.jakarta.transformer=true</microprofile.jvm.args>
+                        <microprofile.jvm.args>${microprofile.jvm.args} -Dorg.eclipse.microprofile.rest.client.tck.timeoutCushion=50000 -Dorg.wildfly.unsupported.skip.jakarta.transformer=true</microprofile.jvm.args>
```

Comment:
This should probably just be 50. In MP REST Client 2.0 it's calculated as seconds, in MP RC 3.0 it's milliseconds.

Reply:
Fixed @jamezp